src/AppBridgeBlock.spec.ts

```diff
@@ -355,7 +355,7 @@ describe('AppBridgeBlockTest', () => {
     test('updateColorPalette with language set', () => {
         setTranslationLanguage('de');
 
-        const frontifyColorPalette = FrontifyColorPaletteDummy.with(COLOR_PALETTE_ID);
+        const frontifyColorPalette = ColorPaletteDummy.with(COLOR_PALETTE_ID);
```

Comment:
Here as well we should update the constant name as well: 
```suggestion
        const colorPalette = ColorPaletteDummy.with(COLOR_PALETTE_ID);
```